packages/docs/package.json

```diff
@@ -50,7 +50,7 @@
     "webfontloader": "^1.6.28"
   },
   "devDependencies": {
-    "@babel/core": "^7.5.5",
+    "@babel/core": "^7.0.0-0",
```

Comment:
Why downgrading here?

Reply:
I'll double check the warnings associated.